cloudman/helmsman/tests/mock_client.py

```diff
@@ -6,21 +6,27 @@
 import yaml
 
 
-class MockHelm(object):
+class MockClient(object):
```

Comment:
I think we should have a new class called MockKubectl or something? These two things don't belong together I think.

Reply:
But we are ultimately just replacing the `run_command`. I guess we could separate them into separate parsers, but in the end, for the mock client the command that matters is `mock_run_command`
everything else is just helpers, so I would argue that the client should be a single mock client. I separated the parsers for `helm` and `kubectl` commands though in https://github.com/galaxyproject/cloudman/pull/105/commits/88ab6e6104dd56e1beb3e2f79eb3ce307a953a3d